src/go/rpk/pkg/cli/cmd/wasm/template/wasm.go

```diff
@@ -9,26 +9,46 @@
 
 package template
 
-const wasmJs = `const { SimpleTransform, PolicyError } = require("@vectorizedio/wasm-api");
+const wasmJs = `const {
+  SimpleTransform,
+  PolicyError,
+  calculateRecordLength,
+  calculateRecordBatchSize,
+} = require("@vectorizedio/wasm-api");
 const transform = new SimpleTransform();
 /* Topics that fire the transform function */
-transform.subscribe([
-  /*\"input topics\"*/
-]);
+transform.subscribe(["produce"]);
 /* The strategy the transform engine will use when handling errors */
 transform.errorHandler(PolicyError.SkipOnFailure);
+/* Auxiliar transform function for records */
+const uppercase = (record) => {
+  const newRecord = {
+    ...record,
+    value: record.value.map((value) => {
+      if (value > 97 && value < 122) {
+        return value - 32;
+      } else {
+        return value;
+      }
+    }),
+  };
+  newRecord.length = calculateRecordLength(newRecord);
```

Comment:
Can't this be calculated automatically by the coproc server?

Reply:
After checking with Alex, yes I'll move this function into coproc engine

Reply:
Done